td-agent/yum/install-test.sh

```diff
@@ -14,11 +14,13 @@ set -exu
 distribution=$(cat /etc/system-release-cpe | awk '{print substr($0, index($1, "o"))}' | cut -d: -f2)
 version=$(cat /etc/system-release-cpe | awk '{print substr($0, index($1, "o"))}' | cut -d: -f4)
 
+ENABLE_UPGRADE_TEST=1
 case ${distribution} in
   amazon)
     case ${version} in
       2)
         DNF=yum
+	ENABLE_UPGRADE_TEST=0
```

Comment:
indent

Reply:
Is it O.K. to skip upgrading test for Amazon Linux 2? Following code seems trying to do it for this platform...

Reply:
@ashie Thanks, I've fixed wrong baseurl, so I've enabled test again.